src/mongocxx/test/spec/monitoring.cpp

```diff
@@ -116,10 +132,36 @@ std::string apm_checker::print_all() {
     return output.str();
 }
 
+/**
+ * @brief Determine whether 'event' is one of the "sensitive" events. `event` must have a
+ *      `command_name() const -> stdx::string_view` method.
+ */
+template <typename Ev>
+static bool is_sensitive_command(const Ev& event) noexcept {
+    static stdx::string_view sensitive_commands[] = {
+        "authenticate",
+        "saslStart",
+        "saslContinue",
+        "getnonce",
+        "createUser",
+        "updateUser",
+        "gopydbgetnonce",
+        "copyDbSaslStart",
+        "copydb",
+    };
+    return std::find(std::begin(sensitive_commands),
+                     std::end(sensitive_commands),
+                     event.command_name())  //
```

Comment:
Is the trailing `//` intentional to break the line?

Reply:
Yeah, it's become a bit of a (weird) habit I use to slap `clang-format` on the wrist to make it do some nicer formatting.

Reply:
Ah good to know!